contracts/staking/stakeManager/StakingNFT.sol

```diff
@@ -0,0 +1,34 @@
+pragma solidity ^0.5.2;
+
+import "openzeppelin-solidity/contracts/token/ERC721/ERC721Full.sol";
+import {Ownable} from "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract StakingNFT is ERC721Full, Ownable {
+    constructor(string memory name, string memory symbol)
+        public
+        ERC721Full(name, symbol)
+    {
+        // solhint-disable-previous-line no-empty-blocks
+    }
+
+    function mint(address to, uint256 tokenId) public onlyOwner {
+        require(
```

Comment:
IMO, this should be checked by client contract before calling this function.

Reply:
`client contract` has no information regarding owned nft's so instead of calling 2 functions `balanceOf` and then `mint` it makes sense to have a single call.